website/blog/2020-02-31-version-60.md

```diff
@@ -0,0 +1,61 @@
+
+## Lean Core Removals
+
+**WebView** and **NetInfo** were previously extracted into separate repositories, and in 0.60 we’ve finished migrating them out of the React Native repository. Additionally, in response to community feedback about new App Store policy, **Geolocation** has been extracted. If you haven’t already, complete your migration by adding dependencies to [react-native-webview](https://github.com/react-native-community/react-native-webview), [@react-native-community/netinfo](https://github.com/react-native-community/react-native-netinfo), and [@react-native-community/geolocation](https://github.com/react-native-community/react-native-geolocation). Maintainers have made more than 100 commits to these repositories since extraction and we’re excited to see the community’s support!
+
+## Native Modules are now Autolinked
+
+The team working on the [React Native CLI](https://github.com/react-native-community/cli) has introduced major improvements to native module linking called [autolinking](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md)! Most scenarios will not require the use of `react-native link` anymore. At the same time, the team overhauled the linking process in general. Let us know what you think using the hashtag [#autolinkingInRN](https://twitter.com/intent/tweet?hashtags=autolinkingInRN).
+
+## Upgrade Helper
+
+[@lucasbento](https://github.com/lucasbento), [@pvinis](https://github.com/pvinis), [@kelset](https://github.com/kelset), and [@watadarkstar](https://github.com/watadarkstar) have built a great tool called [Upgrade Helper](https://react-native-community.github.io/upgrade-helper/) to make the upgrade process simpler. It helps React Native users with brownfield apps or complex customizations to see what's changed between versions. Take a look at the [upgrade from 0.59.9 to 0.60.0](https://react-native-community.github.io/upgrade-helper/?from=0.59.9&to=0.60.0) and try it out today for your upgrade path!
+
+![Upgrade Helper cleanly and easily shows the changes needed to migrate to a different version of React Native](/react-native/blog/assets/0.60-upgrade-helper.png)
+
+## A Note to Library Maintainers
+
+Changes for AndroidX will almost certainly require updates to your library, so be sure to include support soon. If you're not able to upgrade yet, consider checking your library against the jetifier to confirm that users are able to patch your library at build time.
```

Comment:
there should also be a mention that if the library has an iOS native side and is using Pods, they have to change their Pod deps -> https://github.com/facebook/react-native/commit/2321b3f

Reply:
@kelset I asked around and was unable to find examples of this happening, it seems like an edge case.

Reply:
I guess that maybe for libraries is an edge case - but I'm sure some apps will be hit by the BC. Maybe add the link to the commit where you say "some straightforward changes" in the Pods paragraph?